src/main/java/solutions/fairdata/fdp/index/api/filters/TokenFilter.java

```diff
@@ -43,16 +43,18 @@
 @Component
 public class TokenFilter extends OncePerRequestFilter {
 
-    private final String PREFIX = "Bearer ";
+    private static final String PREFIX = "Bearer ";
 
     @Autowired
     private TokenService tokenService;
 
     @Override
     protected void doFilterInternal(HttpServletRequest request, HttpServletResponse response, FilterChain filterChain) throws IOException, ServletException {
         String token = getToken(request);
+        System.out.println("TOKEN: " + token);
         if (token != null) {
             Optional<Authentication> auth = tokenService.getAuthentication(token);
+            System.out.println(auth);
```

Comment:
slf4j logger with a debug statement?

Reply:
Oops... need to find some checker for those 😅 fixed, thanks